reviews/client/src/components/Search.jsx

```diff
@@ -7,14 +7,11 @@ class Search extends React.Component {
 
   render() {
     return (<div>
-      {/* <form>
-        <input type="text" placeholder="Search reviews" value={this.props.value} onChange={this.props.handleSearch}/>
-      </form> */}
-      <div className="search">
-        <span className="fa fa-search"></span>
-        {/* <img src="http://download.seaicons.com/icons/icons8/ios7/512/Very-Basic-Search-icon.png"/> */}
-        <input placeholder="Search reviews" />
-     </div>
+      <form className="search">
+        {/* <span className="fa fa-search"></span> */}
```

Comment:
Probably remove these eventually

Reply:
yes! they're placeholder for some ideas I have for rendering the stupid magnifying glass for the search bar...